src/App.jsx

```diff
@@ -31,17 +31,17 @@ const allScopes = [
 let callbackUri = window.location.href.split('/').slice(0, 4).join('/');
 
 // if the callback uri ends with a slash, remove it
-callbackUri = callbackUri.charAt(callbackUri.length - 1) === '/' ? callbackUri.slice(0, callbackUri.length - 1) : callbackUri;
+callbackUri = callbackUri.endsWith() === '/' ? callbackUri.slice(0, callbackUri.length - 1) : callbackUri;
```

Comment:
Theres a small bug here - should be:

```
callbackUri = callbackUri.endsWith('/') ? callbackUri.slice(0, callbackUri.length - 1) : callbackUri;
```

Reply:
Fixed here: https://github.com/alecchendev/spotify-refresh-token/pull/9/commits/9986b59ecc9bc1f3ca52dd269446b6f97b79424f